app/components/pipeline/Header/index.js

```diff
@@ -158,7 +158,7 @@ class Header extends React.Component<Props, State> {
         })
       },
       {
-        allowed: "pipelineUpdate",
+        always: true,
```

Comment:
Does this look okay on mobile with one option in the menu? 🤔

Reply:
Yeah, okay "enough"